src/stories/Checbox.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { StoryObj, Meta } from "@storybook/react";
+import { Checkbox } from "../../lib/main";
+import { useState } from "react";
+
+const meta = {
+  title: "Components/Checkbox",
+  component: Checkbox,
+  parameters: { layout: "centered" },
+  tags: ["autodocs"],
+  args: {
+    disabled: false,
+    checked: false,
+    className: "",
+    label: "Get updates about Deriv products, services and events.",
+    onChange: () => {},
+    wrapperClassName: "",
+    error: false,
+  },
+  argTypes: {
+    wrapperClassName: {
+      control: {
+        disable: true,
+      },
+    },
+    className: {
+      control: {
+        disable: true,
+      },
+    },
+    checked: {
+      control: {
+        type: "boolean",
+      },
+    },
+    error: {
+      control: {
+        type: "boolean",
+      },
+    },
+    onChange: {
+      control: {
+        disable: true,
+      },
+    },
+    label: {
+      control: {
+        type: "text",
+      },
+    },
+    disabled: {
+      control: {
+        type: "boolean",
+      },
+    },
+  },
+} satisfies Meta<typeof Checkbox>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  name: "Default Checkbox Input",
+  args: {
+    checked: false,
+    onChange: () => {},
+    label: "Get updates about Deriv products, services and events.",
+  },
+  render: (args) => {
+    const [checked, setChecked] = useState(false);
+
+    return (
+      <Checkbox
+        {...args}
+        checked={checked}
+        onChange={() => setChecked(!checked)}
```

Comment:
@aizad-deriv it's not so important to write like this here, but better to get used for this format

```suggestion
        onChange={() => setChecked(previous => !previous)}
```